metrics/cli/lib/sentry/model/sentry_project.dart

```diff
@@ -0,0 +1,15 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+/// A class that represents slugs for the Sentry project.
+class SentryProject {
+  /// The identifier of the Sentry organization.
+  final String organizationSlug;
+
+  /// The identifier of the Sentry project.
+  final String projectSlug;
+
+  /// Creates a new instance of the [SentryProject]
+  /// with the given [organizationSlug] and [projectSlug].
+  SentryProject(this.organizationSlug, this.projectSlug);
```

Comment:
```suggestion
  SentryProject({
    this.organizationSlug,
    this.projectSlug,
  });
```
What do you think? Also, would it be great to throw an argument error if any of the parameters is null?